jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -1,27 +1,29 @@
-from specutils import Spectrum1D
+import pytest
+from astropy import units as u
+from astropy.nddata import CCDData
+from astropy.tests.helper import assert_quantity_allclose
 
-from jdaviz import Application
 from jdaviz.configs.cubeviz.plugins.moment_maps.moment_maps import MomentMap
 
 
-def todo_fix_test_moment_calculation(spectrum1d_cube):
-
-    app = Application()
+@pytest.mark.filterwarnings('ignore:No observer defined on WCS')
+def test_moment_calculation(cubeviz_app, spectrum1d_cube):
+    app = cubeviz_app.app
     dc = app.data_collection
     app.add_data(spectrum1d_cube, 'test')
 
     mm = MomentMap(app=app)
     mm._subset_selected = 'None'
-    # mm.spectral_min = 1.0 * u.m
-    # mm.spectral_max = 2.0 * u.m
     mm._on_data_updated(None)
 
     mm._on_data_selected({'new': 'test'})
     mm._on_subset_selected({'new': None})
 
-    mm.n_moment = 0
+    mm.n_moment = 0  # Collapsed sum, will get back 2D spatial image
     mm.vue_calculate_moment()
 
     assert mm.moment_available
     assert dc[1].label == 'Moment 0: test'
-    assert dc[1].get_object(cls=Spectrum1D, statistic=None).shape == (4, 2, 2)
+
+    result = dc[1].get_object(cls=CCDData)
+    assert_quantity_allclose(result, [[24, 30], [27, 33]] * u.Jy)
```

Comment:
I have no idea if this result is actually correct. This is not the same as a simple sum along the relevant axis. Naively, I would have expected something like this:

```python
>>> a = np.arange(16).reshape((4, 2, 2))
>>> a.sum(axis=0)
array([[24, 28],
       [32, 36]])
```

Reply:
Nevermind... trying to fix this broke Gaussian Smooth. I really don't grok why the shape keeps changing for cubes in this app.